bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -294,15 +294,14 @@ class ModelImporter(
 
     private fun syncReferences(node: INode, nodeData: NodeData) {
         nodeData.references.forEach {
```

Comment:
tip: you can use destructuring like this: `nodeData.references.forEach { (key, value) ->`